src/app/components/Curation/CurationPromo/index.tsx

```diff
@@ -48,16 +48,17 @@ const CurationPromo = ({
     (type === 'photogallery' && `${photoGalleryTranslation}, `);
 
   return (
-    <Promo>
+    <Promo css={styles.promo} className="">
```

Comment:
Don't like having to put an empty classname here, this is basically to appease the TS compiler as Promo is a non-typescript file, the compiler is inferring types. Anyone know a cleaner way, without having to convert?

Reply:
Yea not sure tbh. I've faced this quite a bit with the mixture of TSX and JSX and had to do what you've done here.

Reply:
The other option is to // @ts-expect-error with a description, but this is probably OK for now? (The advantage of the  @ts-expect-error is that if we were to typescript-ify the legacy Promo component, and it was no longer an error, then we would have to remove the comment...